src/explore-education-statistics-admin/src/pages/release/data/ReleaseApiDataSetLocationsMappingPage.tsx

```diff
@@ -100,11 +115,27 @@ export default function ReleaseApiDataSetLocationsMappingPage() {
 
   const navItems: NavItem[] = useMemo(() => {
     return [
+      ...(Object.keys(deletedLocationGroups).length > 0
+        ? [
+            {
+              id: 'deleted-location-groups',
```

Comment:
We've got a naming disparity between this and the filter columns, you've gone with 'deleted' whereas I went with 'mappable' on the basis that in the future you'll be able to map filter columns. Is the plan to be able to map location groups eventually? If so, it would be good to standardise the naming to avoid confusion.

Reply:
I've gone with deleted as I don't think there's any plans to make these mappable. There's a number of cases where mapping isn't really possible e.g. you can't turn a school into an region.

Changes to location groups also aren't going to be common so it's generally more tolerable not to have mappings than it is for filter changes.